content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -0,0 +1,134 @@
+metric type (`counter`, `gauge`, and `histogram`). The `metrics` section
+provides values for the metric dimensions as expressions, and allows you to
+remove or override the existing metric dimensions. You can modify the standard
+metrics using `tags_to_remove` or by re-defining a dimension.
+
+## Before you begin
+
+[Install Istio](/docs/setup/) in your cluster and deploy an application.
+Alternatively, you can set up custom statistics as part of the Istio
+installation.
+
+## Enable custom metrics
+
+Edit the the `EnvoyFilter` to add or modify dimensions and metrics. Then, add
+annotations to all the Istio-enabled pods to extract the new or modified
+dimensions.
+
+1. Locate the `envoy.wasm.stats` extension configuration. The default
+   configuration is in the `configuration` section and typically looks like:
+
+    {{< text yaml >}}
+    {
+        "debug": "false",
+        "stat_prefix": "istio"
+    }
+    {{< /text >}}
+
+1. Modify the configuration section for each instance of the extension
+   configuration. For example, to add `destination_port` and `request_host`
+   dimensions to the standard `requests_total` metric, change the configuration
+   section to look like the following. Istio automatically prefixes all metric
+   names with `istio_`, so omit the prefix from the name field in the metric
+   specification.
+
+    {{< text yaml >}}
+    {
+       "debug": "false",
+       "stat_prefix": "istio",
+       "metrics": [
+          {
+            "name": "requests_total",
+            "dimensions": {
+               "destination_port": "destination.port",
+               "request_host": "request.host"
+            }
+          }
+       ]
+    }
+    {{< /text >}}
+
+1. Apply the following annotation to all injected pods with the list of the
+   dimensions to extract into a Prometheus [time
+   series](https://en.wikipedia.org/wiki/Time_series), using the following
+   command:
+
+    {{< text yaml >}}
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    spec:
+    template:
+        metadata:
+        annotations:
+            sidecar.istio.io/extraStatTags: destination_port,request_host
+    {{< /text >}}
+
+## Verify the results
+
+Use the following command to verify that Istio generates the data for your new
+or modified dimensions:
+
+{{< text bash >}}
+$ kubectl exec <pod-name> -c istio-proxy -- curl 'localhost:15000/metrics’ | grep istio
+{{< /text >}}
+
+For example, in the output, locate the metric `istio_requests_total` and
+verify it contains your new dimension.
+
+## Use expressions for values
+
+The values in the metric configuration are common expressions, which means you
+must double-quote strings in JSON, e.g. "`some_string_value`". Unlike Mixer
+expression language, there is no support for the pipe (`|`) operator, but you
+can be emulate it with the `has` or `in` operator, for example:
+
+{{< text plain >}}
+has(request.host) ? request.host : "unknown"
```

Comment:
Can you give an example of "mixer" expression translated to new syntax?

Reply:
This uses CEL, right? A link would be useful https://opensource.google/projects/cel

Reply:
Done